.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+package-lock.json
```

Comment:
Lock file should be added to git.
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5#:~:text=Yes%2C%20package%2Dlock.,be%20checked%20into%20source%20control.